backend/functions/src/student/functions.ts

```diff
@@ -112,7 +112,7 @@ export const addStudentSurveyResponse = async (
   }
 
   // 0. Check if email is valid cornell.edu email.
-  const emailRegex = /^\w+@cornell.edu$/
+  const emailRegex = /^\w+@cornell\.edu$/
```

Comment:
idr how regex works oop, does this like, make sure emails end with explicitly .edu and not like .education?

Reply:
Probably has to do with the dot matching any character other than line terminators lol

Reply:
ah ok. btw, hi Changyuan!